packages/firebase_crashlytics/README.md

```diff
@@ -62,6 +62,18 @@ apply plugin: 'com.google.gms.google-services'
 apply plugin: 'com.google.firebase.crashlytics'
 ```
 
+4. Add the SDK dependencies in your **app-level** `build.gradle` file (`[project]/android/app/build.gradle`).
+
+```gradle
+dependencies {
+  // Recommended: Add the Firebase SDK for Google Analytics.
+  implementation 'com.google.firebase:firebase-analytics:17.3.0'
```

Comment:
This dependency is not necessary and I believe that the plugin's Pubspec should declare the Android dependencies.

Reply:
> I believe that the plugin's Pubspec should declare the Android dependencies.

What do you mean?

Reply:
I believe the `analytics` dependency is not required for running Crashlytics - it is a recommendation from the Firebase guide. Should we include it in the plugin's README then?

Furthermore, I think that the plugin can add the crashlytics Android dependency, so the user does not have to do that.

Reply:
> Furthermore, I think that the plugin can add the crashlytics Android dependency, so the user does not have to do that.

Are you sure about that? Because: #1153

Reply:
@axel-op No, I am not 😶 I think I have not added Android dependencies in a Flutter app before, however.